src/main/java/it/pagopa/pn/deliverypush/utils/HtmlSanitizerJsonParserDelegate.java

```diff
@@ -0,0 +1,32 @@
+package it.pagopa.pn.deliverypush.utils;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.util.JsonParserDelegate;
+import org.apache.commons.text.StringEscapeUtils;
+import org.owasp.html.PolicyFactory;
+
+import java.io.IOException;
+
+public class HtmlSanitizerJsonParserDelegate extends JsonParserDelegate {
+
+    private final PolicyFactory policy;
+
+
+
+    public HtmlSanitizerJsonParserDelegate(JsonParser d, PolicyFactory policy) {
+        super(d);
+        this.policy = policy;
+    }
+
```

Comment:
Commenta con un caso d'uso l'utilizzo della unescape

Reply:
Il sanificatore, non avendo più la dipendenza della classe PhysicalAddressWriter, non ha più bisogno del refactoring della suddetta classe in metodi statici. Tuttavia, potrebbe essere utile comunque farlo (magari in un altro task)